packages/kbn-legacy-logging/src/schema.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Joi from 'joi';
+
+const HANDLED_IN_KIBANA_PLATFORM = Joi.any().description(
+  'This key is handled in the new platform ONLY'
+);
+
+export interface LegacyLoggingConfig {
+  silent: boolean;
+  quiet: boolean;
+  verbose: boolean;
+  events: Record<string, any>;
+  dest: string;
+  filter: Record<string, any>;
+  json: boolean;
+  timezone?: string;
+  rotate: {
+    enabled: boolean;
+    everyBytes: number;
+    keepFiles: number;
+    pollingInterval: number;
+    usePolling: boolean;
+    pollingPolicyTestTimeout?: number;
+  };
+}
+
+export const legacyLoggingConfigSchema = Joi.object()
+  .keys({
+    appenders: HANDLED_IN_KIBANA_PLATFORM,
+    loggers: HANDLED_IN_KIBANA_PLATFORM,
+    root: HANDLED_IN_KIBANA_PLATFORM,
+
+    silent: Joi.boolean().default(false),
```

Comment:
I thought it was not possible, TIL: we actually got `siblingRef` in config-schema.

Reply:
From what I can see: **legacy platform** doesn't read config values, but pass them to `@kbn/legacy-logging` so it might be okay to move config validation to those `@kbn/legacy-logging` methods that accept config as a function parameter. The main downside of this approach is that the logging config will be validated at a different time than the other config parts.

Reply:
Let's keep it that way until https://github.com/elastic/kibana/issues/81483 is resolved imho.